server.js

```diff
@@ -40,14 +40,15 @@ const plugins = [
     {
         register: require('good'),
         options: {
-            reporters: [{
-                reporter: require('good-console'),
-                events: {
-                    log: '*',
-                    response: '*',
-                    ops: '*'
-                }
-            }]
+            ops: {
+                interval: 30 * 1000
```

Comment:
Not sure what interval is reasonable for this, I don't know how logs are managed/rotated on the server. Took a punt on 30s.